packages/core/src/helpers/parsePluginModuleDefinitions.ts

```diff
@@ -0,0 +1,29 @@
+import { merge } from 'lodash-es';
+
+import type { PluginModuleDefinition } from '@jpmorganchase/mosaic-types';
+
+/**
+ * Filters duplicate plugin module definitions, merging them
+ * together with the user-defined definition (a.k.a. the last
+ * one) taking precedence in cases where there is duplication.
+ *
+ * This is useful for when a plugin is defined in the standard
+ * generator but then also overridden by the user's config.
+ *
+ * @param plugins an array of plugin module definitions
+ * @returns an array of plugin module definitions with duplicates removed
+ */
+export default function parsePluginModuleDefinitions(
+  plugins: PluginModuleDefinition[]
+): PluginModuleDefinition[] {
+  const pluginsByPath = plugins.reduce((acc, plugin) => {
```

Comment:
I guess this is where we add a filter for disabled plugins ?

or is there a better place ?